### PR TITLE
Use angular to sanitize plugin description.

### PR DESCRIPTION
**Pull request by diosmosis**

### CODE REVIEW

File: plugins/CorePluginsAdmin/templates/macros.twig [hunk 1]
```diff
@@ -141,7 +141,7 @@
                         </div>
                         <div class="plugin-desc-text">
 
-                            {{ plugin.info.description|raw|nl2br }}
+                            <div ng-bind-html="{{ plugin.info.description|raw|nl2br|json_encode|e('html_attr') }}"></div>
```
Comment (tsteur) [inline plugins/CorePluginsAdmin/templates/macros.twig]: It seems the only plugin that uses html is custom variables? I'd be tempted to simply not allow HTML in the plugin description.

Comment (diosmosis) [reply plugins/CorePluginsAdmin/templates/macros.twig]: Don't have an opinion personally.

Comment (tsteur) [reply plugins/CorePluginsAdmin/templates/macros.twig]: I'll remove the HTML from custom variables and no longer allow HTML there.

File: plugins/CorePluginsAdmin/templates/macros.twig [hunk 2]
```diff
@@ -150,7 +150,7 @@
                                 'https://matomo.org', 'https://www.matomo.org', 'https://matomo.org/', 'https://www.matomo.org/'
                             ] %}
                                 <span class="plugin-homepage">
-                            <a target="_blank" rel="noreferrer noopener" href="{{ plugin.info.homepage }}">({{ 'CorePluginsAdmin_PluginHomepage'|translate|replace({' ': '&nbsp;'})|raw }})</a>
+                            <a target="_blank" rel="noreferrer noopener" href="{{ plugin.info.homepage|e('url') }}">({{ 'CorePluginsAdmin_PluginHomepage'|translate|replace({' ': '&nbsp;'})|raw }})</a>
```
Comment (tsteur) [inline plugins/CorePluginsAdmin/templates/macros.twig]: `e('url')` cannot be used on a full link, but only on a component. It breaks the URL...

I'll replace it with `html_attr`

**Comment by tsteur** [comment]
@diosmosis see last commit, I fixed the homepage URL link and no longer allow HTML in the description.

**Pull request merged**